src/main/java/mate/academy/springbootintro/model/Category.java

```diff
@@ -0,0 +1,27 @@
+package mate.academy.springbootintro.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import jakarta.persistence.Column;
+import jakarta.persistence.GenerationType;
+import lombok.Data;
+import org.hibernate.annotations.SQLDelete;
+import org.hibernate.annotations.Where;
+
+@Entity
+@SQLDelete(sql = "UPDATE categories SET is_deleted = true WHERE id=?")
+@Where(clause = "is_deleted=false")
+@Data
+@Table(name = "categories")

```

Comment:
```suggestion
@Getter
@Setter
@SQLDelete(sql = "UPDATE categories SET is_deleted = TRUE WHERE id = ?")
@SQLRestriction(clause = "is_deleted = FALSE")
@Table(name = "categories")
```
using Data on entities is generally discouraged, Where is now deprecated